tc-server/src/test/java/com/tc/net/protocol/transport/ServerStackProviderTest.java

```diff
@@ -113,10 +113,8 @@ public void testAttachAlreadyConnectedTransport() throws Exception {
     }
 
     // Now make the attach fail due to IllegalReconnectException, we should force a reconnection rejected exception now
-//    when(harness.attachNewConnection(any(TCConnection.class))).thenThrow(new IllegalReconnectException());
     try {
       serverStackProvider.attachNewConnection(nextConnectionID, connection);
```

Comment:
Does this part of the test have any continued relevance?

Reply:
not really